canonical-web-and-design/ubuntu.com
# Ensure background submit works for marketo modals

## Done

- Fix modal marketo forms to prevent redirect, displaying a thank you message inside the modal

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/cube 
- Select the 'Apply for access' button at the bottom of the page, fill out the form and check on submit the form doesn't redirect, but instead shows a thank you message


## Issue / Card

Fixes #9208
